runtime/afl.c

```diff
@@ -165,7 +171,7 @@ CAMLexport value caml_setup_afl(value unit)
 CAMLprim value caml_reset_afl_instrumentation(value full)
 {
   if (full == Val_true) {
```

Comment:
I think it's possible to end up accidentally building a program which calls this primitive before `caml_setup_afl` has been called. If `caml_afl_area_ptr` is initialised to `NULL` could then be:
```suggestion
  if (full == Val_true && caml_afl_area_ptr) {
```
to eliminate that completely?

Reply:
It doesn't hurt.  Applied in the latest version of this PR.